packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -846,9 +846,14 @@ abstract class SliderComponentShape {
 ///  * [SliderTrackShape] Base component for creating other custom track
 ///    shapes.
 class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const RectangularSliderTrackShape();
+  /// Create a slider track that draws 2 rectangles.
+  const RectangularSliderTrackShape({this.disabledThumbGap = 2.0});
+
+  /// Horizontal spacing, or gap, between the disabled thumb and the track.
```

Comment:
Is there a different gap property for enabled sliders? '

Might be good to explain somewhere: there's ordinarily no gap between the bottom of the thumb and the top of the track. Assuming that's true of course.

Reply:
No gap for enabled sliders. I added some more docs and renamed to disabledThumbGapWidth so its even more clear what this gap is (its not top and bottom)